libraries/chain/db_maint.cpp

```diff
@@ -497,7 +495,7 @@ void database::process_budget()
       if( worker_budget_u128 >= available_funds.value )
```

Comment:
This generates a signed/unsigned comparison warning on gcc 7.4.0, not the fault of this PR though.